src/components/resource/components/edit/coverage-settings/coverage-settings.test.js

```diff
@@ -0,0 +1,153 @@
+import {
+  render,
+  cleanup,
+} from '@testing-library/react';
+import { Form } from 'react-final-form';
+import arrayMutators from 'final-form-arrays';
+import createFocusDecorator from 'final-form-focus';
+
+import Harness from '../../../../../../test/jest/helpers/harness';
+import { coverageStatementDecorator } from '../../../_fields/coverage-statement';
+import EditCoverageSettings from './coverage-settings';
+
+jest.mock('../../../../coverage-date-list', () => () => <div>Coverage date list</div>);
+
+const focusOnErrors = createFocusDecorator();
+
+describe('Given EditCoverageSettings', () => {
+  const handleSectionToggleMock = jest.fn();
```

Comment:
Just a suggestion, it's better to call mock functions started with `mock..` => `mockHandleSectionToggle`. Because in philosophy of Jest it would be more correct and and allows you to forward the function inside the mocked component if necessary